packages/schematics/angular/application/other-files/app.module.ts

```diff
@@ -8,10 +8,10 @@ import { AppComponent } from './app.component';
   declarations: [
     AppComponent
   ],
-  imports: [
+  imports: [<% if (!experimentalIvy) { %>
```

Comment:
Is the `routing` option not supported with Ivy atm?

Reply:
No. I don't think routing is supported.